docs/getting_started.md

```diff
@@ -0,0 +1,141 @@
+
+| Information item(s)      | Description                                                                                                                                   |
+|--------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| ClientId                 | Generated by the ePayment Platform and returned to Integrator                                                                                 |
+| CallbackUrl              | Defined by Integrator, will be the prefix address all Callback requests are sent. Peruse our [Callback](#callbacks) section for more context. |
+| ClientSecret             | A bCrypt secret that is **kept secret at the Integrator** and used to generate access tokens.                                                 |
+| ClientSecret bCrypt Hash | A bCrypt secret hash that is sent to the ePaymentPlatform and configured to the Integrators clientID                                          |
+| Authentication Provider  | Inform the EPP team which Authentication Provider you will be utilizing                                                                       |
+| ISS                      | Once the your profile is set up your will receive the Issuer ID corresponding to the your Integrator or Authentication Provider profile.      |
+| EPP public key           | Is sent by EPP during setup, needed to encrypt parts of requests.                                                                             |
+| Token Requestor Name     | Is sent by EPP during setup, needs to be part of enrolment requests.                                                                          |
+
+## Integration guidelines
+
+This section contains general guidelines for integrating with the EPP.
+
+### Context ID 
+All requests support an `X-Correlation-Id` header which can be used to correlate requests and responses. This is especially useful if you always ensure to set this header to a unique value for each request.
+The `X-Correlation-Id` is returned in the corresponding callback, allowing you an additional mechanism to correlate the callback with the original request. It is **required** to use this header for enabling traceability and support. 
+
+### MessageId
+The system acts idempotent on any `messageId`. It is **required** that you use a robust UUID generator (or similar mechanism) to ensure that each request has a unique `messageId`. 
+
+#### MessageId uniqueness & Callbacks
+EPP creates a UUID that is used as a `messageId` for each callback that is used to distinguish between different requests. This `messageId` is considered to be part of the message exchange between EEP and the Integrator.
+This means that you should not re-use this `messageId` for any other requests.
+
+### Asynchronous retry policy
+Any Asynchronous Requests will be retried if the Response from the Integrator is anything other than `2xx`.
```

Comment:
Coming back to this, this statement alone is correct, but it might be important to mention that a 4xx response from the client will also stop us from retrying,  whereas on 5xx's we will attempt the sending again. Else, if we just say "anything other than 2xx", integrators might build their services to expect reattempts on 4xx.

Reply:
Great clarification, changed wording to reflect it!